apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -244,6 +246,24 @@ export const assignToSection = (
     {includeUserId: true}
   );
   return (dispatch, getState) => {
+    const section = getState().teacherSections.sections[sectionId];
```

Comment:
I like the idea of sending them from redux! I recognize that I'm not fully aware how the edit section dialog actually does the work, and my only worry is that we might end up with duplicate events if we have logging from the component and logging from redux. I can look into it for this case!

Reply:
I _think_ it uses `editSectionProperties` directly. I don't see a duplicate event locally so I think we should be good!

Reply:
Awesome thank you!